src/components/TextField.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'styled-components';
+
+const Text = Styled.div`
+  margin-top: ${(props) => props.mt}px;
+  margin-bottom: ${(props) => props.mb}px;
+  text-align: ${(props) => props.align}
+`;
+
+export default function TextField({
```

Comment:
You don't need this functional component since you just copy the props to the styled component